src/main/java/net/imagej/ops/descriptor3d/Facets.java

```diff
@@ -0,0 +1,5 @@
+package net.imagej.ops.descriptor3d;
+
+public interface Facets {
```

Comment:
There is a convention in ImgLib2, and other places in SciJava code including ImageJ Common, to use the plural as a utility class for working with the singular. For example, `Intervals` provides handy methods for working with `Interval` objects. In ImageJ Common: `Axes` for `Axis`. I like this convention as it is succinct, and once you know it you can infer where to look for such things for additional class hierarchies.

For that reason, I'd avoid `Facet` and `Facets` in favor of something else (`FacetGroup`? Not sure on the purpose of the interface since it has no public API).